src/components/MainPageContent.tsx

```diff
@@ -11,34 +11,34 @@ const axiosMainData = async (offset = 0, limit = 0) => {
   );
 };
 
-type contents = [] | { title: string; _id: string }[];
+export type Content = {
+  title: string;
+  _id: string;
+};
 
 function MainPageContent() {
+  const [contents, setContents] = useState<Content[]>([]);
+  const [isContentEmpty, setIsContentEmpty] = useState<boolean>(false);
+  const contentCount = useRef<number>(0);
   const observeRef = useObserver(() => {
-    addContent();
+    moreContent();
   });
-  const [contents, setContents] = useState<contents>([]);
-  const [noMoreContent, setNoMoreContent] = useState<boolean>(false);
-  const contentAddCount = useRef<number>(0);
-
-  const addContent = useCallback(async (limit = 6) => {
-    if (noMoreContent) return;
-    const { data } = await axiosMainData(
-      contentAddCount.current * limit,
-      limit
-    );
+  const moreContent = useCallback(async (limit = 9) => {
+    if (isContentEmpty) return;
+    const { data } = await axiosMainData(contentCount.current * limit, limit);
+    console.log(data);
     setContents((prevContents) => [...prevContents, ...data]);
-    contentAddCount.current++;
-    if (data.length !== limit) setNoMoreContent(true);
+    contentCount.current++;
+    if (data.length !== limit) setIsContentEmpty(true);
   }, []);
 
   return (
     <>
-      <Flex flexDir="column" pos="relative" p="15px 15px 50px 15px">
+      <Flex flexDir="column" pos="relative" p="15 15 50 15">
```

Comment:
```suggestion
      <Flex flexDir="column" pos="relative" p="15 15 50 15">
```
`p="15 15 50 15"` 이 코드가 동작하지 않는 것 같아 보입니다!! 확인 부탁드립니다!